llvm/test/CodeGen/AArch64/GlobalISel/arm64-irtranslator.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
```

Comment:
Converting a test to generated checks should be done separately (also, this will fail since it now has the pre-function checks, and the generated).

It's probably fine to just leave it manual for now

Reply:
It must be Monday. What do you mean by manual?

Reply:
I mean just don't use update_mir_test_checks like the test is already